File: topics/02-key-value-store/java/src/test/java/dev/reinvent/wheel/t02/KeyValueStoreAppTest.java
package dev.reinvent.wheel.t02;

import static org.junit.jupiter.api.Assertions.*;

import org.junit.jupiter.api.Test;

final class KeyValueStoreAppTest {
    @Test
    void minimalTemplateCompilesAndRuns() {
        assertEquals("hello kv-store\n", KeyValueStoreApp.helloMessage(new KeyValueStoreApp.CliArgs("../fixtures/commands.txt")));
    }
}

File: topics/02-key-value-store/kotlin/src/test/kotlin/dev/reinvent/wheel/t02/KeyValueStoreAppTest.kt
package dev.reinvent.wheel.t02

import kotlin.test.Test

class KeyValueStoreAppTest {
    @Test
    fun minimalTemplateCompilesAndRuns() {
        kotlin.test.assertEquals("hello kv-store\n", KeyValueStoreApp.helloMessage(KeyValueStoreApp.CliArgs("../fixtures/commands.txt")))
    }
}
